fs2-google-pubsub-grpc/src/main/scala/com/permutive/pubsub/consumer/grpc/PubsubGoogleConsumer.scala

```diff
@@ -13,41 +13,45 @@ object PubsubGoogleConsumer {
   /**
     * Subscribe with manual acknowledgement
     *
+    * @param blocker
     * @param projectId    google cloud project id
     * @param subscription name of the subscription
     * @param errorHandler upon failure to decode, an exception is thrown. Allows acknowledging the message.
     */
   final def subscribe[F[_]: Concurrent: ContextShift, A: MessageDecoder](
+    blocker: Blocker,
     projectId: Model.ProjectId,
     subscription: Model.Subscription,
     errorHandler: (PubsubMessage, Throwable, F[Unit], F[Unit]) => F[Unit],
     config: PubsubGoogleConsumerConfig[F]
-  ): Stream[F, Model.Record[F, A]] =
+  ): Stream[F, ConsumerRecord[F, A]] =
     PubsubSubscriber
-      .subscribe(projectId, subscription, config)
+      .subscribe(blocker, projectId, subscription, config)
       .flatMap {
         case internal.Model.Record(msg, ack, nack) =>
           MessageDecoder[A].decode(msg.getData.toByteArray) match {
             case Left(e)  => Stream.eval_(errorHandler(msg, e, ack, nack))
-            case Right(v) => Stream.emit(Model.Record(v, ack, nack))
+            case Right(v) => Stream.emit(ConsumerRecord(v, ack, nack, _ => Concurrent[F].unit))
```

Comment:
Possibly still a WIP? Was hoping you would be adding a new commit before you opened the PR!

Reply:
We can't do manual ack extension in the GRPC one. It's automatic and configured on the builder. Best I can do is noop, unless you have better ideas?

Reply:
Oops, my bad! Unrelated/bike-shedding but I generally prefer `Applicative[F].unit` in case we no longer need the concurrent for it's actual purpose, makes it easier to drop down.